### ISSUE TITLE
FloodUI not persisting config.json

### ISSUE THREAD

**Issue by tomato4** [linuxserver/docker-mods#807]
Every time the transmission is updated, the app looses it's configuration (eg. Common Paths). I tried to extract config.json (in /themes/flood-for-transmission) and bind it as external volume, but the mod just rewrites that. I tried to bind it as only RO, but the app just ignores the config (and in log is `cp: cannot create regular file '/themes/flood-for-transmission/config.json': Read-only file system`) and common paths in settings are empty.

I didn't even find the configuration, where the app copies the config from default to try fix it.

This bug is very annoying, cause I either don't ever update the app or I have to assign common paths again and again.

**Comment by aptalca** [comment]
@tomato4 
can you test this PR mod build and let us know if it works?
`ghcr.io/linuxserver/mods:pull_request_840`
just set `-e DOCKER_MODS=ghcr.io/linuxserver/mods:pull_request_840`

It should save the `config.json` in the config folder and make it persistent.

Thanks

**Comment by tomato4** [comment]
I will look at it no later than at the end of the week and I will let you know, if it is fixed.

**Comment by tomato4** [comment]
Partially the problem is solved. If I set the variables in config and start container, the config is reflected in the settings (meaning if I set common_path, the common path is set in the settings).

However if I change the settings in the app, the settings is NOT saved into the config file (`/themes/flood-for-transmission/config.json`). I'm also getting error in the docker log `cp: cannot remove '/themes/flood-for-transmission/config.json': Resource busy`. So if I want to change the settings permanently, I have to edit the config file directly.
